ontology/pg.ttl

```diff
@@ -0,0 +1,102 @@
+# baseURI: http://biohackathon.org/resource/pg
+
+@prefix : <http://biohackathon.org/resource/pg#> .
+@prefix vg: <http://biohackathon.org/resource/vg#> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix spin: <http://spinrdf.org/spin#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix edam: <http://edamontology.org/> .
+@prefix faldo: <http://biohackathon.org/resource/faldo#> .
+
+<http://biohackathon.org/resource/pg>
+  rdf:type owl:Ontology ;
+  <http://purl.org/dc/terms/license> <http://creativecommons.org/publicdomain/zero/1.0/> ;
+  spin:imports <http://topbraid.org/spin/owlrl-all> ;
+  spin:imports <http://topbraid.org/spin/rdfsplus> ;
+  spin:imports <http://topbraid.org/spin/spinowl> ;
+  owl:versionInfo "Created at the CoVID-19 Hackathon"^^xsd:string ;
+.
+
+:ZoomLevel
+  rdf:type owl:Class ;
+  rdfs:comment "A zoom level of Pangenome."^^xsd:string ;
+  rdfs:label "ZoomLevel"^^xsd:string ;
+  rdfs:subClassOf owl:Thing ;
+.
+:zoomFactor
+  rdf:type owl:DatatypeProperty ;
+  rdfs:comment "The zoom factor of pangenome, which is defined as bin width, generally."^^xsd:string ;
+  rdfs:domain :ZoomLevel ;
+  rdfs:label "factor"^^xsd:string ;
+  rdfs:range xsd:positiveInteger ;
+.
+:components
+  rdf:type owl:ObjectProperty ;
+  rdfs:comment "A zoom level has one or more components."^^xsd:string ;
+  rdfs:domain :ZoomLevel ;
+  rdfs:label "has"^^xsd:string ;
+  rdfs:range :Component ;
+.
+:Component
+  rdf:type owl:Class ;
+  rdfs:comment "A region between Arrows shared by all individuals."^^xsd:string ;
+  rdfs:label "Component"^^xsd:string ;
+  rdfs:subClassOf owl:Thing ;
+.
+:componentEdge
+  rdf:type owl:ObjectProperty ;
+  rdfs:comment "The super property that says two components are linked on the PangenomeSequence."^^xsd:string ;
+  rdfs:domain :Component ;
+  rdfs:label "connects"^^xsd:string ;
+  rdfs:range :Component ;
+.
+:componentRank
+  rdf:type owl:DatatypeProperty ;
+  rdfs:comment "The rank records that step place along its pangenome sequence."^^xsd:string ;
+  rdfs:domain :Component ;
+  rdfs:label "rank"^^xsd:string ;
+  rdfs:range xsd:positiveInteger ;
+.
+:bins
+  rdf:type owl:ObjectProperty ;
+  rdfs:comment "A Component has one or more Bins."^^xsd:string ;
+  rdfs:domain :Component ;
+  rdfs:label "has"^^xsd:string ;
+  rdfs:range vg:Bin ;
+.
+:steps
```

Comment:
I am not sure that `:steps` and `:Step` are required. The `:componentRank` and `:binRank` seems to have the same function. In any case when reading vg rdf and pg rdf, having `pg:Step` and `vg:Step` makes debugging harder than necessary (can easily confuse).

Reply:
The reason why we need `pg:Step` is that whether a path appears on the component depends on path.
If there is a path on the component, positionPercent and inversion Percent have each path in each component. Therefore, we need to specify a path of the component.
Because it is confusing, I will change its name.

Reply:
> The :componentRank and :binRank seems to have the same function. 
Should I change the name of `:componentRank` or `:binRank`, or annotate they have the same function?

Reply:
I think it is good the way it is.